### ISSUE TITLE
Please add to F-Droid

### ISSUE THREAD

**Issue by RasheedAZ** [cashubtc/eNuts#273]
Thanks for making this app :) Can you please submit to the F-Droid store for inclusion?

**Comment by KKA11010** [comment]
> Thanks for making this app :) Can you please submit to the F-Droid store for inclusion?

Thank you for opening this issue. I Would love to submit the repo to the F-Droid store but this might be a bigger task because we started the project by using the managed workflow provided by [expo](https://expo.dev/).

Expo abstracts away many aspects of the native development, providing a simplified workflow. It may pose challenges when submitting to platforms like F-Droid and I have to dive into it to find out more.

**Comment by sime** [comment]
The alternative is to host your own F-Droid repository: https://f-droid.org/docs/Setup_an_F-Droid_App_Repo/

**Comment by RasheedAZ** [comment]
> The alternative is to host your own F-Droid repository: https://f-droid.org/docs/Setup_an_F-Droid_App_Repo/

Not really. Hosting your own F-Droid repo is not an alternative to being on F-Droid. I think that can belong in a separate issue.

An app being on the F-Droid store means it is verified to be fully Free and Open Source Software (FOSS) without any closed source binary blobs and dependencies. In many cases the app is fully reproducible.

Third party repos have no such requirements and can even contain closed source software.
